tinyagentos/routes/store_install.py

```
@@ -799,38 +802,46 @@ async def install_app(request: Request):
-                            "detail": (
-                                "The manifest on disk was modified after the initial "
-                                "signature verification. This may indicate post-verification "
-                                "tampering. Rebuild the catalog or reinstall the app from "
-                                "a trusted source."
-                            ),
-                            "install_id": install_id,
-                        },
-                        status_code=403,
-                    )
+                return _verify_sig(on_disk, stored_sig, _store_pub)
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)